### ISSUE TITLE
Add fuel trucks

### ISSUE THREAD

**Issue by Rabadash94** [CleverRaven/Cataclysm-DDA#70776]
### Is your feature request related to a problem? Please describe.

The game's lack of parked, crashed, or surrounded by zombies (happy to bash vehicles for some reason) fuel trucks is alarming.

### Solution you would like.

Add fuel trucks. They should be able to be found in various situations: simply parked, damaged, with spills, on fire, and, of course, not always empty. What's the worst that could happen?

### Describe alternatives you have considered.

Add gas trucks too, perhaps.

### Additional context

Fuel stations don't fill themselves. That's unrealistic.

**Comment by PGR-14** [comment]
We should have them spawn very rarely or next to gas stations.
Also, when gas interacts w/ fire, does it explode?

**Comment by kasanryukin** [comment]
The military use some converted hummvees as fuel carriers.  Something with a trio of 200L drums should be easy enough, but the fuel would be limited to JP8. 
I find this interesting and want to mock up something while we wait on multitile stuff.
